packages/environment-check/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@sap-ux/environment-check",
-    "version": "0.1.4",
+    "version": "0.1.5",
```

Comment:
Please leave the version number as-is. Once you are done, you add a changeset and the version number will be updated automatically.

Reply:
Updated, thanks @tobiasqueck